srcpkgs/wineasio/template

```diff
@@ -1,30 +1,32 @@
 # Template file for 'wineasio'
 pkgname=wineasio
-version=0.9.2
-revision=7
-archs="i686"
-repository="nonfree"
+version=1.0.0
+revision=1
+archs="i686 x86_64"
```

Comment:
At least allow it to build for musl ;)

But I'd just leave it without any restrictions, the dependency on `wine-devel` already restricts it.

Reply:
Oh cool, I didn't realize that it wouldn't fail in a extra gross way.